resource-management/test/resourceRegionMgrSimulator/handlers/regionNodeEvents.go

```diff
@@ -86,11 +87,12 @@ func (re *RegionNodeEventHandler) SimulatorHandler(rw http.ResponseWriter, r *ht
 	if r.URL.Path == InitPullPath || r.URL.Path == SubsequentPullPath {
 		var nodeEvents simulatorTypes.RegionNodeEvents
 		var count uint64
+		var crv types.TransitResourceVersionMap
```

Comment:
since we are not going to have lw and pull models ( for comparison purpose or so ) in aggregator to regionMgr, the pull model handler can removed from the code base. will remove from the api code change.